lib/middleware/notFound.js

```diff
@@ -0,0 +1,4 @@
+/* eslint-disable no-console */
```

Comment:
```suggestion
module.exports = (req, res, next) => {
  res.status(404).send({ error: 'Not Found' });
  next();
};
```
Current code is producing an error on localhost:7890. See above example. I would also recommend checking out the docs on custom [middleware](https://expressjs.com/en/guide/writing-middleware.html) in express.